packages/odyssey-react-mui/src/index.ts

```diff
@@ -68,6 +68,7 @@ export * from "./CssBaseline";
 export * from "./createShadowRootElement";
 export * from "./createUniqueId";
 export * from "./Dialog";
+export * from "./labs/Drawer";
```

Comment:
I know I did this for Switch, on accident. :( But, we probably shouldn't export things in labs from here. We should probably ensure consumers are importing from the labs directory as a reminder that they using a non-finalized component